shell/platform/windows/flutter_windows_engine.cc

```diff
@@ -796,4 +795,27 @@ void FlutterWindowsEngine::OnApplicationLifecycleEnabled() {
   lifecycle_manager_->BeginProcessingClose();
 }
 
+void FlutterWindowsEngine::OnWindowStateEvent(HWND hwnd,
+                                              WindowStateEvent event) {
+  lifecycle_manager_->OnWindowStateEvent(hwnd, event);
+}
+
+std::optional<LRESULT> FlutterWindowsEngine::ProcessExternalWindowMessage(
+    HWND hwnd,
+    UINT message,
+    WPARAM wparam,
+    LPARAM lparam) {
+  if (lifecycle_manager_) {
+    switch (message) {
```

Comment:
Would it be better to remove this `switch` and just let the lifecycle manager decide which messages it responds to? I don't feel strongly about this

Reply:
That is probably a better idea if we want to also respond to `WM_CLOSE` in the future, or make similar additions.